shared/nav.native.js

```diff
@@ -208,17 +209,17 @@ const mapDispatchToProps = (dispatch: Dispatch, ownProps: OwnProps) => ({
   },
 })
 
-const dismissKeyboardOnPathChange = lifecycle({
```

Comment:
this dismisses the keyboard on more than a path change. i think attaching a lifecycle thing to a logical thing like this is likely not going to be long lived as other logic goes into there. we could have multiple lifecycle events in compose but that might also be more confusing